castle/cms/interfaces/controlpanel.py

```diff
@@ -290,6 +290,39 @@ class ISiteConfiguration(Interface):
         default=False
     )
 
+    cdn_alternate_domain = schema.TextLine(
+        title=u'Configure alternate CDN domain',
+        required=False
+    )
+
+    cdn_alternate_port = schema.TextLine(
+        title=u'Configure alternate CDN port',
+        required=False,
+    )
+
+    cdn_alternate_path = schema.TextLine(
+        title=u'Configure alternate CDN path',
+        required=False,
+    )
+
+    cdn_allow_js = schema.Bool(
+        title=u'Include JS in CDN Config',
```

Comment:
The title should be changed to be more direct, ie something like: `Enable CDN URL for JS resources`

This should be done for the css and image urls as well.